tools/bug-report/pkg/bugreport/bugreport.go

```diff
@@ -61,8 +63,16 @@ func Cmd(logOpts *log.Options) *cobra.Command {
 		Use:          "bug-report",
 		Short:        "Cluster information and log capture support tool.",
 		SilenceUsage: true,
-		Long: "This command selectively captures cluster information and logs into an archive to help " +
-			"diagnose problems. It optionally uploads the archive to a GCS bucket.",
+		Long: "bug-report selectively captures cluster information and logs into an archive to help " +
+			"diagnose problems and optionally uploads the archive to a GCS bucket. \n" +
+			"Proxy logs can be filtered using:\n" +
+			"  --include|--exclude namespace1,.../deployment1,.../pod1,.../container1,.../label1=value1,.../annotation1=value1,...\n" +
```

Comment:
what is ...?

Reply:
namespace1,namespace2... etc. i only added namespace2 otherwise the string will be very long.

Reply:
even after this explanation I am not sure how to use it. Maybe we can replace ... with ns1 if its for length. Specifically, `namespace2.../deployment1` is extremely hard to reason about

Reply:
@howardjohn That was done in the code, the snippet is outdated. See https://github.com/istio/istio/pull/27424/files#diff-06d0741f47caf05de9e051ec9ad25778R74

Reply:
I am looking at the new code in my latest comment.